numpyro/distributions/transforms.py

```diff
@@ -1530,3 +1618,8 @@ def _transform_to_softplus_lower_cholesky(constraint):
 @biject_to.register(constraints.simplex)
 def _transform_to_simplex(constraint):
     return StickBreakingTransform()
+
+
+@biject_to.register(constraints.zero_sum)
+def _transform_to_zero_sum(constraint):
+    return ZeroSumTransform(constraint.event_dim).inv
```

Comment:
I think this shouldn't have the `.inv` here.

```python
>>> from numpyro.distributions.transforms import biject_to
>>> from numpyro.distributions.constraints import zero_sum
>>> from jax import random
>>> 
>>> transform = biject_to(zero_sum(1))
>>> transform(random.normal(random.key(7), [3]))
Array([-0.7128072 , -0.14349887], dtype=float32)

```

Reply:
good catch!

Reply:
youre right it definitely shouldnt be there - I just fixed this. question, should one of the tests have been catching that? I didn't see anything pop up with the test cases I added

Reply:
It might be worth adding another test that checks that `biject_to` actually bijects to the desired constraint. Maybe for another PR?

Reply:
I think thats a good call, I'll call this resolved for now then. Thank you!